scheduler_failover_controller/command_runner/command_runner.py

```diff
@@ -29,11 +29,12 @@ def _run_local_command(self, base_command):
         return True, output
 
     def _run_ssh_command(self, host, base_command):
+	print(" The base_command is " + str(base_command))
         self.logger.debug("Running command as SSH command")
         if base_command.startswith("sudo"):
             command_split = ["ssh", "-tt", host, base_command]
         else:
-            command_split = ["ssh", host, base_command]
+            command_split = ["ssh" , "-i", "/home/sshuser/.ssh/hdinsight/ssh_key", host, base_command]
```

Comment:
I am not sure that this ssh key location is good - default is better for this purpose.
Better approach will be create configuration parameter for ssh key location.